src/components/firstProfile/interests/Interests.tsx

```diff
@@ -8,7 +8,10 @@ import { LanguageItem } from './LanguageItem'
 
 const LOCAL_STORAGE_KEY = 'userPreferences'
 
-const Interests = () => {
+interface InterestsProps {
+  showAboutMeFirst?: boolean
```

Comment:
Since this variable is boolean please rename it into something like isAboutMeShown or shouldShowAboutMe

Reply:
The variable has been renamed to isAboutMeShown as suggested